en/installconfig-storage.md

```diff
@@ -1,79 +1,75 @@
-Title: Storage | MAAS
+Title: Storage
+TODO:  Describe the Storage web UI page
+       Explain how to perform actions: LVM, RAID, Bcache, modify filesystems
+       Bug tracking: https://pad.lv/1636933
+       Bug tracking: https://pad.lv/1698891
+       Bug tracking: https://pad.lv/1698895
+       The 'block devices' and 'partitions' pages have been removed from the menu. They should be reviewed. If worthy, they should be moved to the CLI section
+       Re the above, partitions *are* block devices (?)
+table_of_contents: True
 
 
 # Storage
 
-MAAS has the ability to configure any storage layout during node deployment.
-MAAS doesn't just do simple partitioning - it supports complex storage layouts,
-including setting up and configuring Bcache, RAID, and LVM. It also offers
-fine-grained control over the creation, deletion, formatting and mounting of
-both block devices and partitions (see [Block devices][block-devices] and
-[Partitions][partitions]).
+The final storage configuration that a deployed node uses can be influenced
+significantly. MAAS supports traditional disk partitioning as well as more
+complex options such as LVM, RAID, and Bcache. UEFI is also supported.
```

Comment:
I wanted to include UEFI somewhere in the introduction since there is a section about it. Suggestions?